scram-tools.file/tools/professor2/professor2.xml

```diff
@@ -8,4 +8,6 @@
 <use name="yoda"/>
 <use name="eigen"/>
 <runtime name="PATH" value="$PROFESSOR2_BASE/bin" type="path"/>
+<runtime name="PYTHONPATH" value="$PROFESSOR2_BASE/lib/python3.9/site-packages" type="path"/>
+<runtime name="LD_LIBRARY_PATH" value="$LIBDIR" type="path"/>
```

Comment:
this is also not needed. `$PROFESSOR2_BASE/lib` will be in LD_LIBRARY_PATH in cmssw env. So please remove this line

Reply:
Thanks! I was confused indeed since I need to add those paths manually to get things work if I just build professor2... 

Was going to ask but you already pointed out, thanks!